add Dockerfile with Gaeul service on Ubuntu 18.04

xhaakon opened the pull request:
Run the resulting image with
```
  docker run -ti --cap-add SYS_ADMIN -v /sys/fs/cgroup:/sys/fs/cgroup:ro
    --device /dev/video{your-device-number}
```

xhaakon committed:
add Dockerfile with Gaeul service on Ubuntu 18.04

Run the resulting image with

  docker run -ti --cap-add SYS_ADMIN -v /sys/fs/cgroup:/sys/fs/cgroup:ro
    --device /dev/video{your-device-number}

xhaakon commented:
@justinjoy I'm a bit confused by the structure of this repository. `master` points to tag `1.1.6`, but there are also commits tagged `1.1.7`, `1.1.8` and `1.1.9` which are newer than the master branch. Is there any purpose behind this layout?

(This MR is against `master`)

justinjoy commented:
> @justinjoy I'm a bit confused by the structure of this repository. `master` points to tag `1.1.6`, but there are also commits tagged `1.1.7`, `1.1.8` and `1.1.9` which are newer than the master branch. Is there any purpose behind this layout?
> 
> (This MR is against `master`)

Ah. Thank you for letting me know. I just forgot to push `master`. Now I am done.

xhaakon committed:
add Dockerfile with Hwangsae relay service on Ubuntu 19.04

Run the resulting image with

 docker run -ti --cap-add SYS_ADMIN  -v /sys/fs/cgroup:/sys/fs/cgroup:ro

xhaakon committed:
add Dockerfile with Hwangsae recorder service on Ubuntu 19.04

Run the resulting image with

 docker run -ti --cap-add SYS_ADMIN  -v /sys/fs/cgroup:/sys/fs/cgroup:ro
